### PR TITLE
Fix: add missing rollup enums

### LINKED ISSUE #43: Add missing enums for rollups
As per, the following screenshot, `show_unique` and `show_original` are missing from the rollup enum values

image

There is also another bug where the boolean value for a formula can be null but the deserializer does not account for it.

### PR DESCRIPTION
**Pull request by AnkurSheel**
Fixes #43

### CODE REVIEW

File: Src/Notion.Client/Models/Database/Properties/RollupProperty.cs [hunk 1]
```diff
@@ -70,6 +70,12 @@ public enum Function
         Max,
 
         [EnumMember(Value = "range")]
-        Range
+        Range,
+
+        [EnumMember(Value = "show_original")]
+        ShowOriginal,
+
+        [EnumMember(Value = "show_unique")]
```
Comment (KoditkarVedant) [inline Src/Notion.Client/Models/Database/Properties/RollupProperty.cs:78]: @AnkurSheel Thank you for raising the PR. One thing though I checked the API document but I did not see the `show_unique` enum. Rest of the changes LGTM.

https://developers.notion.com/reference/database#rollup-configuration.

**Comment by KoditkarVedant** [comment]
@AnkurSheel can you also create a issue regarding the same so it will help others when they face the issue.

File: Src/Notion.Client/Models/Database/Properties/RollupProperty.cs (hunk 1 above)
Comment (AnkurSheel) [reply Src/Notion.Client/Models/Database/Properties/RollupProperty.cs:78]: It is one of the options when configuring rollups
image

**Comment by AnkurSheel** [comment]
> @AnkurSheel can you also create a issue regarding the same so it will help others when they face the issue.

Done.

**Pull request merged**